### PR TITLE
Portfolio mobile - Completion [ day -2 ] 

### PR DESCRIPTION
**Pull request by roniy68**
**Added Sections :** 
- Recent Works
- About Me
**Used in CSS: **
- Grid
- Flex Box

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -77,3 +93,197 @@ a {
...
+  align-items: center;
+  justify-content: space-between;
+  height: 333px;
+  margin: 60px 0;
+  padding: 0 25px;
+  background-image: url(photo/illustration-\ about\ me-\ mobile.svg);
+  background-position: center;
+  background-size: cover;
+  background-repeat: no-repeat;
+
```
Comment (Ugboaja-Uchechi) [inline css/style.css]: - [x] Great job with your design 💪 . Kindly make sure the `background-image` in the `About Me` section displays 🙏 

Figma Design | Your Design
------------- | -------------
image  |  image

File: css/style.css [hunk 2]
```diff
@@ -7,58 +7,74 @@
...
+/* .portfolio {
   background: white;
-  background-image: url('../images/background.png');
+  background-image: url('../images/header-illsutration-mobile@2x.svg');
   background-size: cover;
   background-position: center;
   background-repeat: no-repeat;
   border-radius: 0;
-}
+} */
```
Comment (Ugboaja-Uchechi) [inline css/style.css]: - [x] Kindly remove all commented code 🙏 . This will help make your codebase neater and easier to read.

**Review by Ugboaja-Uchechi — CHANGES_REQUESTED** [review]
Hi @roniy68, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- Linters are passing ✔️ 
- README is descriptive 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/style.css (hunk 1 above)
Comment (roniy68) [reply css/style.css]: resolved the issue

File: css/style.css (hunk 2 above)
Comment (roniy68) [reply css/style.css]: cleaned the comments

**Comment by roniy68** [comment]
**Solved The Issues :**
-  background of about-me section.
-  removed all the comments.

File: css/style.css [hunk 3]
```diff
@@ -77,3 +82,188 @@ a {
...
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+  height: 333px;
+  margin: 60px 0;
+  padding: 0 25px;
+  background-image: url("../images/about-me-top.svg"), url("../images/about-me-bottom.svg");
+  background-repeat: no-repeat;
+  background-position: top right, bottom left;
+}
```
Comment (abdulhamiid) [inline css/style.css]: - [x] As mentioned in the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) that you must stick to the design as much as possible (e.g., font, colors, images, text, margins)

- [x] Please make sure that your design is exactly the same as the one in [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). according to the [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md) as Figma design is very detailed.

Figma design 👇 
bg22

Your design 👇 
bg11

- [x] Please let us remove the (left and right) margins surrounding the background image as shown in the diagram above ☝️

**Review by abdulhamiid — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @roniy68, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, I really appreciate the effort you put in! but there are still some which haven't been addressed yet.

almost-there

## Highlights 🌟

- ✔️ Descriptions are very descriptive
- ✔️ Linters checks are passing
- ✔️ Use GitHub Flow
- ✔️ Meaningful commit messages.

## Required Changes ♻️
_Please kindly check the comments under the review._

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/style.css (hunk 3 above)
Comment (roniy68) [reply css/style.css]: ok is there any other changes nedded ? I have one more review left

Comment (abdulhamiid) [reply css/style.css]: No, that's all 😊  👍

Comment (roniy68) [reply css/style.css]: ok I have fixed the background ... Can you review one more time ?

**Comment by roniy68** [comment]
Fixed the padding issue of About me background.

File: css/style.css (hunk 3 above)
Comment (abdulhamiid) [reply css/style.css]: Hi, @roniy68, you have to submit your project in your dashboard for re-reviews

**Review by amejid — APPROVED** [review]
Hi @roniy68, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Descriptive PR :heavy_check_mark: 
- Correct use Github flow :heavy_check_mark: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lancelot-SO — COMMENTED** [review]
- [ ] Please, kindly make sure your background image shows. Thank you.
- [ ] You should decide to use js to make it dynamic also. 

**Further Improvement**
======================
- [ ] You should also use a media query to make the website responsive.

haha

File: css/style.css [hunk 4]
```diff
@@ -7,58 +7,62 @@
   font-family: "Inter", sans-serif;
```
Comment (kazaneza) [inline css/style.css:7]: - [ ] In my opinion you should have more than two fonts-family for backup

**Comment by kazaneza** [comment]
**Almost There**
==========================================
almost

***However some changes I have suggested t you in the review section***

**Pull request merged**